CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # CHANGELOG
 
-## Release 2.6.0 (in development)
+## Release 2.6.1 (development in progress)
```

Comment:
No, you want a new line for 2.6.1 above the line for 2.6.0 (and remove the `in development` part from the latter)

Reply:
okay